opssc-agent/src/notifier.ts

```diff
@@ -29,11 +29,14 @@ export class Notifier {
     logger.debug('WS URL: %s', config.websocketUrl);
     try {
       this.ws = new WebSocket(config.websocketUrl);
-      this.ws.on('open', function open() {
-        logger.info('Opened ws');
+      this.ws.on('open', () => {
+          logger.info('Opened ws');
```

Comment:
Looks like the indents are mixed (`opssc-api-server/src/config.ts` also seems to have lines with 4-space indents). I'd suggest fixing them with some lint tool.